rom/dev/src/flow/cold_reset/fmc_alias.rs

```diff
@@ -157,34 +179,79 @@ impl FmcAliasLayer {
     /// report_error is called. This prevents a race condition where the SoC
     /// reads FW_ERROR_NON_FATAL immediately after the mailbox transaction
     /// fails, but before caliptra has set the FW_ERROR_NON_FATAL register.
-    fn download_image(env: &RomEnv) -> CaliptraResult<ManuallyDrop<MailboxRecvTxn>> {
+    fn process_mailbox_commands(env: &RomEnv) -> CaliptraResult<ManuallyDrop<MailboxRecvTxn>> {
```

Comment:
This seems to be the place where we can add the parameter consistency check to meet the FIPS task in ROM.